MovieApp/Screens/Home/Views/ListCell/ListMovieCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ListMovieCell.swift
+//  MovieApp
+//
+//  Created by MinhDev on 10/5/21.
+//
+
+import UIKit
+
+class ListMovieCell: UITableViewCell {
+	
+	@IBOutlet private weak var imageListMovieCell: UIImageView!
```

Comment:
@minh5896 : Add // MARK: -
```suggestion
        // MARK: - Outlets
	@IBOutlet private weak var imageListMovieCell: UIImageView!
```